assets/tests/consolidate_1_1_tests.yaml

```diff
@@ -666,12 +915,17 @@ tests:
           - object: document:4
             relation: owner
             user: user:duck
-        assertions:
+        checkAssertions:
           - tuple:
               object: document:1
               relation: viewer
               user: user:aardvark
             expectation: true
+          - tuple:
+              object: document:1
+              relation: viewer
+              user: user:badger
+            expectation: true
```

Comment:
This assertion and this one

```
- tuple:
              object: document:3
              relation: viewer
              user: user:cheetah
            expectation: true
```

are pretty much the same in the current setup of tuples. 

Can you replace line 904 with `relation: writer`?